pkg/vfs/backup.go

```diff
@@ -63,23 +63,23 @@ func Backup(m meta.Meta, blob object.ObjectStorage, interval time.Duration, skip
 		}
 		if now := time.Now(); now.Sub(last) >= interval {
 			var iused, dummy uint64
+			_ = m.StatFS(ctx, meta.RootInode, &dummy, &dummy, &iused, &dummy)
 			if interval <= time.Hour {
-				_ = m.StatFS(ctx, meta.RootInode, &dummy, &dummy, &iused, &dummy)
 				if iused > 1e6 {
 					logger.Warnf("backup metadata skipped because of too many inodes: %d %s; "+
 						"you may increase `--backup-meta` to enable it again", iused, interval)
 					continue
 				}
 			}
 			if st := m.SetXattr(ctx, 0, key, []byte(now.Format(time.RFC3339)), meta.XattrCreateOrReplace); st != 0 {
-				logger.Warnf("setxattr inode 1 key %s: %s", key, st)
+				logger.Warnf("setxattr inode 0 key %s: %s", key, st)
```

Comment:
It will actually set xattr on inode 1 (the root inode), see `baseMeta.checkRoot`.

Reply:
OK, updated. I thought it was a typo